packages/suite-desktop-core/e2e/tests/settings/electrum.test.ts

```diff
@@ -7,8 +12,14 @@ import { onTopBar } from '../../support/pageActions/topBarActions';
 import { onSettingsPage } from '../../support/pageActions/settingsActions';
 import { onDashboardPage } from '../../support/pageActions/dashboardActions';
 
+let electronApp: ElectronApplication;
+let window: Page;
+let localDataDir: string;
+
 testPlaywright.describe.serial('Suite works with Electrum server', () => {
     testPlaywright.beforeAll(async () => {
+        ({ electronApp, window, localDataDir } = await launchSuite());
+        rmDirRecursive(localDataDir);
```

Comment:
I'd move rmDirRecursive into launchSuite, called conditionally based on some arg. I suggest: `wipe`. Would be the same pattern as with launching device emu